pkg/controllers/hub/trafficmanagerbackend/controller_integration_test.go

```diff
@@ -111,12 +129,58 @@ func updateTrafficManagerProfileStatusToTrue(ctx context.Context, profile *fleet
 	Expect(k8sClient.Status().Update(ctx, profile)).Should(Succeed())
 }
 
+// validateTrafficManagerBackendMetricsEmitted validates the trafficManagerBackend status metrics are emitted and are emitted in the correct order.
+func validateTrafficManagerBackendMetricsEmitted(registry *prometheus.Registry, wantMetrics ...*prometheusclientmodel.Metric) {
```

Comment:
Your UTs use a slice directly, the second parameter could just be a slice. But it's up to you if you want to keep code consistency.